check_abi.sh

```diff
@@ -99,14 +105,22 @@ function abi_check() {
 # TODO: If ABI_BASE_URL is not provided, need to build from last tag/merge (as industrial_ci)
 test -z "$ABI_BASE_URL" && echo -e $(colorize YELLOW "For ABI check, please specify ABI_BASE_URL variable") && exit 1
 
-test "$TRAVIS" == true && abi_install
-
-# fetch and extract old abi from $ABI_BASE_URL
-mkdir -p "${ABI_TMP_DIR}/old"
-travis_run --display "Download and extract base ABI" \
+if [ "$ABI_BASE_URL" == "generate" ] ; then
+	test "$TRAVIS" == true && abi_install
+	travis_run abi_check \
+			"${ROS_WS}/install/lib" "${ROS_WS}/install/include" \
+			"${ROS_WS}/install/lib" "${ROS_WS}/install/include"
+elif [ "$TRAVIS_PULL_REQUEST" == true ]; then
+	# For a pull request, actually perform the abi check
+	test "$TRAVIS" == true && abi_install
```

Comment:
Do we need `== true`?

Reply:
Yes, shell doesn't have boolean (or other typed) variables. For example, the following snippet will echo `yes` although the variable is false:
```shell
foo=false; test "$foo" && echo yes
```